From 4ecd6ef73fd0c40db467e5795741a4344889765d Mon Sep 17 00:00:00 2001
From: Alan Marchesan <justcavalli@gmail.com>
Date: Fri, 5 Feb 2016 06:30:21 +0100
Subject: [PATCH 11/28] Disable 'Keep audio device alive' to speed up
 suspension when turning off the device and Kodi is
 playing something

---
 system/settings/settings.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system/settings/settings.xml b/system/settings/settings.xml
index 76c9a33..d51c5bd 100644
--- a/system/settings/settings.xml
+++ b/system/settings/settings.xml
@@ -2550,7 +2550,7 @@
         </setting>
         <setting id="audiooutput.streamsilence" type="integer" label="421" help="34111">
           <level>1</level>
-          <default>1</default>
+          <default>0</default>
           <constraints>
             <options>audiostreamsilence</options>
           </constraints>
-- 
1.7.10.4

